plugin/build.gradle.kts

```diff
@@ -107,11 +107,11 @@ pluginBundle {
     tags = listOf("ktlint", "kotlin", "linting")
 
     (plugins) {
-        register("ktlintPlugin") {
+        "ktlintPlugin" {
             id = "org.jlleitschuh.gradle.ktlint"
             displayName = "Ktlint Gradle Plugin"
         }
-        register("ktlintIdeaPlugin") {
+        "ktlintIdeaPlugin" {
```

Comment:
😕 Are you sure this is right? `plugins` is a `NamedDomainObjectContainer`. Calling `register` is the way you're supposed to add elements to the collection. Does this actually work as is?

Reply:
actually it complained on `register() {..}` after update.

Also check kotlin sample here: https://plugins.gradle.org/docs/publish-plugin - it uses approach that uses in this PR.

Reply:
Huh, you're right. I asked Eric about this in the Gradle Community Kotlin-DSL channel.
I think that the `gradlePlugin.plugins` block creating the elements in the `pluginBundle.plugins` `NamedDomainObjectCollection`.